ui/src/components/btn-toggle/QBtnToggle.js

```diff
@@ -66,11 +66,13 @@ export default Vue.extend({
   methods: {
     __set (value, opt) {
       if (this.readonly !== true) {
-        if (this.clearable === true && value === this.value) {
-          this.$emit('input', null, null)
-          this.$emit('clear')
+        if (this.value === value) {
```

Comment:
I feel so dumb when you do this kind of thing with my code LOL

Reply:
You did the ground-work, which is what matters the most! The rest is not important.